pkg/api/v1/pod/util.go

```diff
@@ -188,6 +188,7 @@ func VisitPodSecretNames(pod *v1.Pod, visitor Visitor) bool {
 	return true
 }
 
+// visitContainerSecretNames returns true if it finds reference to Secret in the container, otherwise false
```

Comment:
This is not an accurate description of what this function is doing. This returns true unless the visitor returned false when invoked with a secret reference.